src/db.ts

```diff
@@ -8,18 +8,26 @@ let db: Connection | undefined;
 export async function getDB() {
 	if (db) return db;
 
+	// Require ssl in production
+	const extra_opts =
```

Comment:
This should be camelCase, other then this LGTM, I'll test this locally now.

Reply:
Done (force pushed to update the commit)